src/ignitus-UserInterfaceBook/Components/Organisms/interfaceOpportunitySideCard/Components/index.tsx

```diff
@@ -7,6 +7,6 @@ export const interfaceOpportunitySideCard = () => (
   <Interface>
     <Heading2>Opportunity Side Card</Heading2>
     <hr />
-  <InfoCard location="Australia" commitment="Full-time" salary="24" />
+    <InfoCard location="Australia" commitment="Full-time" salary="24" />
```

Comment:
I dont know why fixing lint errors modified this but I am committing to whatever fix it did.